src/txs/Tx.tsx

```diff
@@ -174,7 +184,7 @@ function Tx<TxValues>(props: Props<TxValues>) {
   const getNativeMax = () => {
     if (!balance) return
     const gasAmount = gasFee.denom === token ? gasFee.amount : "0"
-    return calcMax({ balance, rate, cap, gasAmount }).max
+    return calcMax({ balance, rate: shouldTax ? rate : "0", cap, gasAmount }).max
```

Comment:
Can we make `useTaxRate()` return `0` to avoid this force value?

Reply:
Implemented. Good suggestion!